spec/tree_spec.rb

```diff
@@ -5,10 +5,25 @@
   it 'should be a Class' do
     expect(described_class.is_a? Class).to eq true
   end
+  it 'should create an instance of Tree with correct attributes' do
+    tree = Tree.new(5, 1, 0, true)
+    expect(tree.height).to eq(5);
+    expect(tree.age).to eq(1);
+    expect(tree.apples).to eq(0);
+    expect(tree.alive).to eq(true);
+  end
 end
 
-describe 'Fruit' do
+describe Fruit do
+  it 'should be a Class' do
+    expect(described_class.is_a? Class).to eq true
+  end
+  
 end
 
-describe 'Apple' do
+describe Apple do
+  it 'apple should be a an instance of Apple and subclass of Fruit' do
+    apple = Apple.new()
+    expect(apple).to be_a_kind_of(Fruit)
+  end
```

Comment:
If you find yourself putting 'and' in the description of your test, then you should consider writing two tests instead.

Reply:
I've changed it to be two tests, one for Apple < Fruit, and one for apple to be_an_instance_of(Apple)